public/sprite.svg

```diff
@@ -3,7 +3,7 @@
 <symbol id="light" viewBox="0 0 24 24" fill="currentColor"><path d="M20.993 13.313a6 6 0 01-7.306-7.306 7 7 0 107.306 7.306z"></path><path fill-rule="evenodd" d="M4.5 8.25a.5.5 0 01.5.5v1.5a.5.5 0 01-1 0v-1.5a.5.5 0 01.5-.5z" clip-rule="evenodd"></path><path fill-rule="evenodd" d="M3.25 9.5a.5.5 0 01.5-.5h1.5a.5.5 0 010 1h-1.5a.5.5 0 01-.5-.5zM7.5 3a.5.5 0 01.5.5v2a.5.5 0 01-1 0v-2a.5.5 0 01.5-.5z" clip-rule="evenodd"></path><path fill-rule="evenodd" d="M6 4.5a.5.5 0 01.5-.5h2a.5.5 0 010 1h-2a.5.5 0 01-.5-.5z" clip-rule="evenodd"></path></symbol>
 <symbol id="dark" viewBox="0 0 24 24" fill="currentColor"><path d="M12 16.5A4.505 4.505 0 017.5 12c0-2.481 2.019-4.5 4.5-4.5s4.5 2.019 4.5 4.5-2.019 4.5-4.5 4.5z"></path><path fill-rule="evenodd" d="M12 3a.5.5 0 01.5.5v2a.5.5 0 01-1 0v-2A.5.5 0 0112 3zM18 12a.5.5 0 01.5-.5h2a.5.5 0 010 1h-2a.5.5 0 01-.5-.5zM3 12a.5.5 0 01.5-.5h2a.5.5 0 010 1h-2A.5.5 0 013 12zM12 18a.5.5 0 01.5.5v2a.5.5 0 01-1 0v-2a.5.5 0 01.5-.5zM18.354 5.646a.5.5 0 010 .708l-1.5 1.5a.5.5 0 01-.708-.708l1.5-1.5a.5.5 0 01.708 0zM7.854 16.146a.5.5 0 010 .708l-1.5 1.5a.5.5 0 01-.708-.708l1.5-1.5a.5.5 0 01.708 0zM5.646 5.646a.5.5 0 01.708 0l1.5 1.5a.5.5 0 11-.708.708l-1.5-1.5a.5.5 0 010-.708zM16.146 16.146a.5.5 0 01.708 0l1.5 1.5a.5.5 0 01-.708.708l-1.5-1.5a.5.5 0 010-.708z" clip-rule="evenodd"></path></symbol>
 <symbol id="BTC" viewBox="0 0 24 24" fill="currentColor"><path fill-rule="evenodd" d="M13.425 6.432c1.983.19 3.538.778 3.71 2.528.117 1.276-.438 2.035-1.355 2.463 1.481.359 2.382 1.202 2.196 3.072-.227 2.343-2.035 2.952-4.62 3.08l.004 2.42-1.522.002-.004-2.42c-.166-.002-.34 0-.519.003-.238.003-.484.006-.731-.001l.004 2.42-1.52.001-.004-2.42-3.044-.058.256-1.768s1.15.024 1.129.012c.423-.002.549-.293.58-.485l-.008-3.878.012-2.76c-.046-.288-.248-.634-.87-.644.033-.03-1.115.001-1.115.001L6 6.38l3.12-.005-.004-2.37 1.571-.002.004 2.37c.304-.008.603-.005.906-.003l.3.002-.005-2.37L13.422 4l.003 2.432zm-2.92 4.46l.076.002c.926.04 3.67.155 3.673-1.457-.004-1.532-2.339-1.482-3.423-1.46-.129.003-.24.006-.327.005v2.91zm.129 4.75l-.134-.005v-2.91c.097.002.218 0 .359-.002 1.282-.015 4.145-.05 4.132 1.494.014 1.597-3.218 1.468-4.357 1.423z" clip-rule="evenodd"></path></symbol>
-<symbol id="sats" viewBox="0 0 24 24" fill="currentColor"><path fill-rule="evenodd" d="M16.25 8.66h-10v-1.5h10v1.5zM12.75 2.25v2.455h-1.5V2.25h1.5zM12.75 17.795v2.455h-1.5v-2.455h1.5zM16.25 12.75h-10v-1.5h10v1.5zM16.25 16.84h-10v-1.5h10v1.5z" clip-rule="evenodd"></path></symbol>
+<symbol id="sats" viewBox="0 0 30 30" fill="none"><path d="M8 10H22" stroke="#323232" stroke-width="2"/><path d="M15 7V4" stroke="#323232" stroke-width="2"/><path d="M15 26V23" stroke="#323232" stroke-width="2"/><path d="M8 15H22" stroke="#323232" stroke-width="2"/><path d="M8 20H22" stroke="#323232" stroke-width="2"/></symbol>
```

Comment:
```suggestion
<symbol id="sats" viewBox="0 0 30 30" fill="none"><path d="M8 10H22" stroke="currentColor" stroke-width="2"/><path d="M15 7V4" stroke="currentColor" stroke-width="2"/><path d="M15 26V23" stroke="currentColor" stroke-width="2"/><path d="M8 15H22" stroke="currentColor" stroke-width="2"/><path d="M8 20H22" stroke="currentColor" stroke-width="2"/></symbol>
```

Reply:
Oh yeah sure good catch! I should've looked at the code not just copy it from Figma. 🤦‍♂️ Thanks!